### PR TITLE
the form can add data to the database

### PR DESCRIPTION
**Pull request by HilbertSpitzer**
#67

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -79,6 +79,22 @@ server.get('/organisations/login', (req, res) => {
   res.render('organisations_login')
 })
 
+//
+//
+server.post('/add-event', function (req, res) {
+  // get the data from the request
```
Comment (mattlub) [inline src/server.js:85]: this route should require authentication! I just remembered

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

**Comment by mattlub** [comment]
fix the conflict but it's 👍

**Comment by mattlub** [comment]
habibi

**Comment by jbesraa** [comment]
@HilbertSpitzer what if not all the fields are filled? you can end up with undefined variables

**Comment by HilbertSpitzer** [comment]
You've got 15 minutes tough guy! Can you manage? @esraajb

**Comment by jbesraa** [comment]
well done joudy 👍

**Pull request merged**